dagger-common/src/main/java/io/odpf/dagger/common/serde/typehandler/primitive/LongHandler.java

```diff
@@ -49,14 +49,27 @@ public Object parseSimpleGroup(SimpleGroup simpleGroup) {
     }
 
     @Override
-    public Object getArray(Object field) {
+    public Object parseRepeatedObjectField(Object field) {
         List<Long> inputValues = new ArrayList<>();
         if (field != null) {
             inputValues = (List<Long>) field;
         }
         return inputValues.toArray(new Long[]{});
     }
 
+    @Override
+    public Object parseRepeatedSimpleGroupField(SimpleGroup simpleGroup) {
+        String fieldName = fieldDescriptor.getName();
+        ArrayList<Long> deserializedValues = new ArrayList<>();
```

Comment:
let's keep variable naming consistent with other classes? longArray?

Reply:
Will do this

Reply:
Fixed via commit https://github.com/odpf/dagger/pull/148/commits/a28474cef99f9a873be2ff405380bd27f2a9f0b7